README.md

```diff
@@ -97,6 +85,53 @@ updates automatically.
 
 To further enhance your Yomitan experience, it's worth [integrating with Anki](./docs/anki-integration.md), a spaced-repetition flashcard program to help solidify the words you encounter.
 
+## Changes
+
+> While we have made some substantial changes, the majority of the extension's functionality is thanks to hard work of [FooSoft](https://foosoft.net/) and many other incredible open source contributors from 2016-2023.
+
+Major changes:
+
+- 💪 Completed the Manifest V2 → V3 transition, [read why here](https://developer.chrome.com/blog/resuming-the-transition-to-mv3/)!
+- ⚡ Switched to using ECMAScript modules and npm-sourced dependencies.
+- 🔧 Implemented an end-to-end CI/CD pipeline.
+- 🧪 Switched to standard testing frameworks, vitest and playwrights.
+
+In addition, we have made important bug fixes and minor enhancements:
+
+- ✨ Improved dictionary import speeds by 2x~10x.
+- 📚 Added functionality to import/export multiple dictionaries, enabling portability across devices.
+- 🕵️‍♂ And [more](https://github.com/themoeway/yomitan/pulls?q=is%3Apr+is%3Amerged+-label%3Aarea%2Fdependencies+-label%3Akind%2Fmeta).
+
+Since the owner requested forks be uniquely named, we have chosen a new name, _Yomitan_ (_-tan_ is an honorific used for anthropomorphic moe characters).
+
+## Contributing
+
+🚀 **Dip your toes into contributing by looking at issues with the label [good-first-issue](https://github.com/themoeway/yomitan/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-issue).**
+
+Since this is a distributed effort, we **highly welcome new contributors**! Feel free to browse the [issue tracker](https://github.com/themoeway/yomitan/issues), and read our [contributing guidelines](./CONTRIBUTING.md).
+
+Here are some ways anyone can help:
+
+- Try using the Yomitan dev build. Not only do you get the latest features, you can help uncover bugs and give feedback to developers early on.
+- Document any UI/UX friction in Github Issues. We're looking to make Yomitan more accessible to non-technical users.
+- All the issues in `area/bug` older than 2 months need help reproducing. If anything interests you, please try to reproduce it and report your results. We can't easily tell if these issues are one-off, have since resolved, or are no longer relevant.
```

Comment:
```suggestion
- All the issues in `area/bug` older than 2 months need help reproducing. If anything interests you, please try to reproduce it and report your results. We can't easily tell if these issues are one-off, have since been resolved, or are no longer relevant.
```